docs/howto-inhibit-upgrade.md

```diff
@@ -25,82 +41,127 @@ name = 'check_system_arch'
 [
   {
-    "stamp": "2018-09-25T19:20:32.143709Z",
+    "stamp": "2019-04-16T15:20:32.143709Z",
     "hostname": "leapp",
     "actor": "check_system_arch",
     "topic": "system_info",
     "context": "904b0170-cfe7-4217-81d3-a259550e73c1",
     "phase": "NON-WORKFLOW-EXECUTION",
     "message": {
-      "hash": "483ac0e46c99535e893ab30cf0d95774830f8406f1633594c97469cf16773ec3",
-      "data": "{\"details\": \"This process can only be executed on x86_64 systems\", \"result\": \"Fail\", \"severity\": \"Error\", \"solutions\": \"There is no current solution for this problem\", \"summary\": \"Unsupported arch!\"}"
+      "hash": "dcdf1679b6fd4c2e21bc4e4ed6585df75cd46aeea90a53ca76f469a2a1aa50d2",
+      "data": "{\"audience\": [\"sysadmin\"], \"detail\": \"{\\\"summary\\\": \\\"Upgrade process is only supported on x86_64 systems.\\\"}\", \"flags\": [\"inhibitor\"], \"renderers\": {\"html\": \"<h2 class=\\\"report-title\\\">{{ title }}</h2><p class=\\\"report-summary\\\">{{ summary }}</p>\", \"plaintext\": \"{{ title }}\\n{{ summary }}\"}, \"severity\": \"high\", \"title\": \"Unsupported arch\"}"
     },
     "type": "Inhibitor"
   }
 ]
 ```
 
-This is all that an actor needs to do in order to verify if some condition is present on the system and inhibit the upgrade process based on that check.
+Or to inspect closely the message.data filed, we could use `jq` tool:
+```sh
+snactor run CheckSystemArch --verbose --print-output | jq '.[] | .message.data | fromjson'
+{
+  "audience": [
+    "sysadmin"
+  ],
+  "detail": "{\"summary\": \"Upgrade process is only supported on x86_64 systems.\"}",
+  "flags": [
+    "inhibitor"
+  ],
+  "renderers": {
+    "html": "<h2 class=\"report-title\">{{ title }}</h2><p class=\"report-summary\">{{ summary }}</p>",
+    "plaintext": "{{ title }}\n{{ summary }}"
+  },
+  "severity": "high",
+  "title": "Unsupported arch"
+}
+```
 
-After all the system checks are executed by different actors, an existing actor named [VerifyCheckResults](https://github.com/oamg/leapp-repository/tree/master/repos/system_upgrade/el7toel8/actors/verifycheckresults) is scheduled to run in the Leapp upgrade workflow. If some [Inhibitor](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/models/reports.py) message was generated by some previous execution of another actor in any previous phase of the workflow, like the sample one we just wrote, the following output will be displayed to the user:
+This is all that an actor needs to do in order to verify if some condition is
+present on the system and inhibit the upgrade process based on that check.
+
+After all the system checks are executed by different actors, an existing actor
+named [VerifyCheckResults](https://github.com/oamg/leapp-repository/tree/master/repos/system_upgrade/el7toel8/actors/verifycheckresults)
+is scheduled to run in the Leapp upgrade workflow. If some [Report](https://github.com/oamg/leapp/blob/master/leapp/reporting/__init__.py)
```

Comment:
Link to https://leapp.readthedocs.io/en/latest/pydoc/leapp.reporting.html#leapp.reporting.Report instead.